src/registrar/models/public_contact.py

```diff
@@ -29,7 +29,8 @@ class ContactTypeChoices(models.TextChoices):
 
     def save(self, *args, **kwargs):
         """Save to the registry and also locally in the registrar database."""
-        if hasattr(self, "domain"):
+        skip_epp_save = kwargs.pop("skip_epp_save", False)
```

Comment:
Can possibly remove this control if we're not using it

Reply:
I think .pop removes it from the kwarg list and just returns if it existed or not. This is being used as a flag to determine if we want to do the entire EPPLib saving process or not

Reply:
My point is that it seems that we're always saving (never passing skip_epp_save=False)

Reply:
@rachidatecs We are on line 1227 of domain.py: `public_contact.save(skip_epp_save=True)` as part of the `_get_or_create_public_contact()` function:
![image](https://github.com/cisagov/getgov/assets/141044360/a7870367-20f9-46ad-b13e-6b9bd40c2a88)